src/layers/StaticImageLayer.js

```diff
@@ -0,0 +1,96 @@
+import { CompositeLayer, COORDINATE_SYSTEM } from '@deck.gl/core';
+import XRLayer from './XRLayer';
+
+import { padColorsAndSliders } from './utils';
+
+const defaultProps = {
+  pickable: false,
+  coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
+  sliderValues: { type: 'array', value: [], compare: true },
+  channelIsOn: { type: 'array', value: [], compare: true },
+  colorValues: { type: 'array', value: [], compare: true },
+  colormap: { type: 'string', value: '', compare: true },
+  domain: { type: 'array', value: [], compare: true },
+  translate: { type: 'array', value: [0, 0], compare: true },
+  scale: { type: 'number', value: 1, compare: true },
+  loader: {
+    type: 'object',
+    value: {
+      getRaster: () => [],
+      vivMetadata: { imageHeight: 0, imageWidth: 0, dtype: '<u2' }
+    },
+    compare: true
+  }
+};
+
+function scaleBounds({ imageWidth, imageHeight, translate, scale }) {
+  const left = translate[0];
+  const top = translate[0];
```

Comment:
Should these be the same? Isn't translate `[top, left]`?

Reply:
Yes.